HOW_TO.md

```diff
@@ -103,22 +107,113 @@ any template .yaml files.
+7. Add workload method name (workload_pod/workload_vm) to environment_variables_dict['workloads'] in [environment_variables.py](benchmark_runner/main/environment_variables.py)
+8. Create workload folder in the [templates](benchmark_runner/common/template_operations/templates) directory.  Create the following files in that directory:
+   1. Add workload_data_template for configuration parameters, e.g. [stressng_data_template.yaml](benchmark_runner/common/template_operations/templates/stressng/stressng_data_template.yaml).  
+   2. The data template is structured as discussed [below](#data-template).
+   3. Add workload pod and VM custom resource template inside [internal_data](benchmark_runner/common/template_operations/templates/stressng/internal_data)
+9. Add workload folder path in [MANIFEST.in](MANIFEST.in), add 2 paths: the workload path to 'workload_data_template.yaml' and path to 'internal_data' Pod and VM template yaml files
+   ```
+     include benchmark_runner/common/template_operations/templates/stressng/*.yaml
+     include benchmark_runner/common/template_operations/templates/stressng/internal_data/*.yaml
+   ```
+10. Add tests for all new methods you write under `tests/integration`.
+11. Update the golden unit test files as described [above](#add-new-workload-modify-parameters-to-workload-or-change-parameters-for-any-ci-job)
+12. For test and debug workload, need to configure [environment_variables.py](benchmark_runner/main/environment_variables.py)
+13. Fill parameters: workload, kubeadmin_password, pin_node_benchmark_operator, pin_node1, pin_node2, elasticsearch, elasticsearch_port
+14. Run [main.py](/benchmark_runner/main/main.py)  and verify that the workload run correctly
+15. The workload can be monitored and checked through 'current run' folder inside the run workload flavor (default flavor: 'test_ci')
+16. Open Kibana url and verify workload index populate with data:
+17. Create the workload index: Kibana -> Hamburger tab -> Stack Management -> Index patterns -> Create index pattern -> workload-results -> timestamp -> Done
+18. Verify workload-results index is populated: Kibana -> Hamburger tab -> Discover -> workload-results (index) -> verify that there is a new data
+
+## Add new custom workload to benchmark runner
+This section also applies to modifying an existing workload, including
+any template .yaml files.
+1. git clone https://github.com/redhat-performance/benchmark-runner
+2. cd benchmark-runner
+3. Install prerequisites (these commands assume RHEL/CentOS/Fedora):
+   - dnf install python3-pip
+4. Create workload Dockerfile, example:
+   ```
+   FROM quay.io/centos/centos:stream8
+   Shell/Python that run workload
+   Result in Json (redirect to stdout)
+   Wrap Json output with begin/end workload stamp
+   start_stamp='@@~@@START-WORKLOAD@@~@@'
+   end_stamp='@@~@@END-WORKLOAD@@~@@'
+   ```
```

Comment:
Can you point to your vdbench Dockerfile for a full example?

Reply:
Actually vdbench output is csv. In this short example I show a simple workload Json output.